src/store/ChartState.ts

```diff
@@ -395,7 +399,10 @@ class ChartState {
             this.stxx.chart.panel.yAxis.drawCurrentPriceLabel = !this.endEpoch;
             this.stxx.preferences.currentPriceLine = !this.endEpoch;
             this.stxx.isAutoScale = this.settings && this.settings.isAutoScale !== false;
+            this.stxx.preferences.whitespace = this.settings?.whitespace || this.chartStore.whitespace;
+            this.stxx.minimumLeftBars = this.settings?.minimumLeftBars || this.chartStore.defaultMinimumBars;
             this.stxx.draw();
+            if (this.has_updated_settings) this.stxx.home();
```

Comment:
If `settings` got updated after chart initialization, chart will scroll back to the latest tick.